build-ps/percona-server.spec

```diff
@@ -495,10 +500,15 @@ datadir=$(/usr/bin/my_print_defaults server mysqld | grep '^--datadir=' | sed -n
 %else
 /sbin/chkconfig --add mysql
 %endif
-# We need this because we don't provide my.cnf on centos 6
-# and the default system one doesn't have info needed
-%if 0%{?rhel} < 7
-if [ $1 -eq 1 ]; then
+if [ ! -f /etc/my.cnf ]
```

Comment:
sed wouldn't be executed as ﻿⁠⁠⁠⁠/usr/bin/my_print_defaults mysqld|grep -c log-error﻿⁠⁠⁠⁠would return 1